templates/mongodb.conf.erb

```diff
@@ -1,35 +1,36 @@
 #mongodb.conf - generated from Puppet
 
-#System Log
-
+<% config_hash = {} -%>
+<% if @syslog || @logpath || @logappend || @system_logrotate || !@verbose.nil? || @verbositylevel -%>
+<% config_hash['systemLog'] = {} -%>
 <% if @syslog -%>
-systemLog.destination: syslog
+<% config_hash['systemLog']['destination'] = 'syslog' -%>
```

Comment:
syslog param sets destination to syslog, but there is no param to set syslogFacility ...